proxstar/__init__.py

```diff
@@ -188,18 +191,19 @@ def vm_details(vmid):
             usage_check=usage_check)
     else:
         abort(403)
```

Comment:
I think you can return `abort(403)`, but that's minor

Reply:
Checked and that works. Change made in next commit